rudder-agent/debian/rules

```diff
@@ -112,7 +112,7 @@ binary-arch: install
 #	dh_perl
 	dh_makeshlibs
 	dh_installdeb
-	dh_shlibdeps -- --ignore-missing-info
+	LD_LIBRARY_PATH="$(CURDIR)/debian/tmp/opt/rudder/lib:$${LD_LIBRARY_PATH}" dh_shlibdeps -- --ignore-missing-info
```

Comment:
Can you add a comment here to say why and for which distro we did that, to easily cleanup or spot future problems ?
Thanks